testsuite/integration-arquillian/tests/base/testsuites/suite.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+GROUP=$1
+if [ "$GROUP" == "" ]; then
+  echo "Usage: suite.sh database|fips"
+  exit
+fi
+
+DIR=`readlink -f $0 | xargs dirname`
+cd $DIR
+
+TESTSUITE_FILE="$GROUP-suite"
+
+TESTS=`cat $TESTSUITE_FILE`
+
+echo $TESTS | sed 's/ /,/g'
```

Comment:
Missing quotes, and overall should be simplified:

```suggestion
GROUP="${1:?Usage: suite.sh database|fips}"
DIR=`readlink -f "$0" | xargs dirname`
sed 's/ /,/g' < "$DIR/$GROUP-suite"
```

Reply:
Nitpick, and I disagree that your version is better, rather it's just more compressed.

Reply:
Missing quotes are nitpick?

Reply:
Quotes everywhere